src/components/Header.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Header() {
+    return(
+        <div>
+            <header className="bg-sky-200 w-screen h-28 flex justify-between">
+                <img src="./RH-logo-with-text.svg" alt="ReHub logo with text" className="w-56 mb-2 max-[1024px]:hidden"/>
```

Comment:
instead of path in src use object and import them. 
e.g. import logo from "./RH-logo-with....";